lib/buildkite/collector/uploader.rb

```diff
@@ -36,6 +36,8 @@ def self.traces
     ]
 
     def self.configure
+      Buildkite::Collector.logger.debug("hello from RSpec thread")
```

Comment:
Is this one meant to be here?

Reply:
Yes.

Since we use the main RSpec thread and spawn a new Writer thread and the log contains thread id (`tid`), I think it would be useful to know which one is the main RSpec thread and Writer thread?

```
2022-05-19T07:43:52.795221000Z pid=1977 tid=7840 DEBUG: hello from RSpec thread
2022-05-19T07:43:53.839428000Z pid=1977 tid=22120 DEBUG: transmitted record_results ["uuid-1"]
```

Reply:
Nice!